keyboards/mmkeyboard/class60/soldered/soldered.c

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+
+#ifdef AUDIO_ENABLE
+float caps_on[][2] = SONG(CAPS_LOCK_ON_SOUND);
+float caps_off[][2] = SONG(CAPS_LOCK_OFF_SOUND);
+
+float num_on[][2] = SONG(NUM_LOCK_ON_SOUND);
+float num_off[][2] = SONG(NUM_LOCK_OFF_SOUND);
+
+float scroll_on[][2] = SONG(SCROLL_LOCK_ON_SOUND);
+float scroll_off[][2] = SONG(SCROLL_LOCK_OFF_SOUND);
+
+bool led_update_kb(led_t led_state) {
```

Comment:
`keyboard.json` specifies indicators, but the implementation here overrides the behaviour without that functionality. The expectation here would be that it also has to toggle the declared pins.

Reply:
My interpretation was that this code simply hooks into the generally provided logic, but you're saying that it replaces/overrides this. Is there an easy way of just also triggering the default behaviour? As the intention here is to hook into the LED code for playing sounds, not changing the LED behaviour.

Reply:
Or is there another hook that provides led status updates?

Reply:
Looks like you should be able to do <https://github.com/qmk/qmk_firmware/pull/22972/files#r1650092537>.